ras_party/views/business_view.py

```diff
@@ -86,6 +86,8 @@ def put_business_attributes_ce(sample):
 @business_view.route('/businesses/search', methods=['GET'])
 def get_party_by_search():
     query = request.args.get('query', '')
+    page = int(request.args.get('page', default=1))
+    limit = int(request.args.get('limit', default=100))
 
-    response = business_controller.get_businesses_by_search_query(query)
-    return jsonify(response)
+    businesses, total_business_count = business_controller.get_businesses_by_search_query(query, page, limit)
+    return {'businesses': businesses, 'total_business_count': total_business_count}
```

Comment:
You need to jsonify the return value I think.  I believe it does some stuff for you under the hood like setting the mimetypes and status codes

Reply:
Good catch thanks